packages/react-i18n/docs/react_i18n_schema.md

```diff
@@ -0,0 +1,284 @@
+# Introduction <!-- omit in toc -->
+
+React I18n is a schema comprised of a structure of nested key-value pairs (KVPs). This document captures the nuances of the constraints placed on the schema by the different levels of concern.
+
+<!-- Created using "Markdown All in One" extension for VS Code -->
+- [History](#history)
+  - [Version 1.0](#version-10)
+- [Versioning](#versioning)
+- [Basic Structure](#basic-structure)
+- [Pluralization](#pluralization)
+  - [Reserved pluralization keys](#reserved-pluralization-keys)
+  - [Invalid pluralization keys](#invalid-pluralization-keys)
+  - [Detecting pluralization contexts](#detecting-pluralization-contexts)
+- [Serialization](#serialization)
+  - [Serialization to JSON](#serialization-to-json)
+    - [Comments](#comments)
+  - [Serialization to JSON+Comments](#serialization-to-jsoncomments)
+- [Comparison with Rails I18n](#comparison-with-rails-i18n)
+    - [Root locale key](#root-locale-key)
+    - [Pluralization](#pluralization-1)
+    - [Interpolation](#interpolation)
+
+# History
+
+## Version 1.0
+
+React I18n's schema is loosely based on the [Rails I18n](https://guides.rubyonrails.org/i18n.html) schema.
```

Comment:
Is this necessary with the comparison section in the TOC? Since that section starts with the same sentence.

Reply:
Removed.